app/helper_test.go

```diff
@@ -254,7 +254,7 @@ func (me *TestHelper) CreatePost(channel *model.Channel) *model.Post {
 		UserId:    me.BasicUser.Id,
 		ChannelId: channel.Id,
 		Message:   "message_" + id,
-		CreateAt:  model.GetMillis() - 10000,
+		CreateAt:  time.Now().UnixNano(),
```

Comment:
I don't think this is valid: `CreateAt` is everywhere a unit in milliseconds, and lots of code relies on interpreting that as such. Creating one with a value that is effectively a huge number of milliseconds is likely to break things in an unexpected way.

Reply:
Indeed. It broke 2 other tests and I decided to revert and fix it inside the test. Reverted.